test/helpers/valueSetMapper.js

```diff
@@ -0,0 +1,52 @@
+/**
+ *
```

Comment:
Might be helpful to have some description here explaining why we need to do this. I'm inferring that the Executor expects the ValueSets in a different format from what FHIR provides, so even if we something about that written here it would be super.